src/doc/src/reference/config.md

```diff
@@ -1331,6 +1331,13 @@ Controls whether or not progress bar is shown in the terminal. Possible values:
 
 Sets the width for progress bar.
 
+#### `term.progress.taskbar`
```

Comment:
The [example config](https://doc.rust-lang.org/cargo/reference/config.html#configuration-format) would also need to be updated